### DISCUSSION TITLE
how to register for github education pack if university did not provide any educational email for students and teachers

### DISCUSSION THREAD

**Discussion by mehergcufly** [community/community#137522] [GitHub Education]
### Hi everyone,

how to register for github education pack if university did not provide any educational email for students and teachers

**Comment by andrefqms** [comment]
Hi @mehergcufly 

Take a look at [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) from GitHub Blog, regarding not having an educational email it states the following:

If you don’t have a school-issued email, follow the prompts to fill out additional information.You’re still eligible even if you only have a personal Gmail address, as long as you can provide alternative documentation to verify your current student status.

If the problem persists I'd suggest you launch a ticket over to Github Education's Support Engineers regarding this matter. You can do so [here](https://support.github.com/contact/education).

**Reply by MauroGutierrez1985** [reply]
@andrefqms there is no way to add additional information. After write the name of my college this appears:
![imagen_2024-09-03_152518583](https://github.com/user-attachments/assets/9a7564a2-418f-4dab-96c8-436cd4f1ac28)

and i cant see an option to submit additional document

**Reply by andrefqms** [reply]
If your school provides academic email to students, you will need to add it to your GitHub account and verify it.
Are you using a gmail account ?

They've updated the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. There's a lot of information there. I've included the section that I feel would best assist.

[I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
[What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
[My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

**Comment by MauroGutierrez1985** [comment]
hi @mehergcufly i have exactly the same problem.

**Reply by mehergcufly** [reply]
hi bro
yes this is a problem for students that wanted to use GitHub Student pack but their universities and colleges don't provide Educational emails to them.

**Comment by mehergcufly** [comment]
My university does not provide any educational/academic email to students
and teachers. They just asked us for our personal phone numbers and our
gmail account to be mentioned in their records. I tried to verify my
student status on github by providing my University portal screen short but
github did not consider it then i tried 2nd time by providing the digital
copy of my transcript but github did not consider it too.

So what do I do to verify my student status on github ???

**Comment by max-programming** [comment]
I am kind of in the same situation. This year I am at another university for my B.E. degree and for some reason I see this

![WhatsApp Image 2024-09-07 at 16 20 52_8c610915](https://github.com/user-attachments/assets/0af2c7e6-a73e-485a-ac54-3d071ba7629e)

I asked the teachers and they told us they don't provide the email to students :(

**Comment by mehergcufly** [comment]
That's weird.

**Reply by JohnDelatto** [reply]
Not all university has the infastructure or funding to manage technology administrative tasks like creating domain related email address or even virtual campus tour or library access

Kindly please ensure your support is provided for this Support Requested to address less infastructure issues

Thanks
